components/event-system/queue.js

```diff
@@ -0,0 +1,89 @@
+const HOW_MANY_PRIORITIES_ARE_THERE_NITRO = 6
+const eventQueue = Array.from(Array(HOW_MANY_PRIORITIES_ARE_THERE_NITRO), () => [])
```

Comment:
I didn't know `Array.from` took a mapping function as a second argument. Nifty!

Reply:
yep, ``Array.from(Array(HOW_MANY_PRIORITIES_ARE_THERE_NITRO), () => [])`` would be 1 character shorter than ``Array(HOW_MANY_PRIORITIES_ARE_THERE_NITRO).fill(0).map(() => [])``